capirca/lib/paloaltofw.py

```diff
@@ -750,10 +752,27 @@ def __str__(self):
     rulebase = etree.SubElement(vsys_entry, "rulebase")
     security = etree.SubElement(rulebase, "security")
     rules = etree.SubElement(security, "rules")
+    tag = etree.Element("tag")
+
+    tag_num = 0
 
     # pytype: disable=key-error
     # pylint: disable=unused-variable
     for (header, pa_rules, filter_options) in self.pafw_policies:
+      tag_name = None
+      if header.comment:
+        comment = " ".join(header.comment).strip()
+        if comment:
+          tag_num += 1
```

Comment:
It appears we're incrementing this without checking against a limit. Can PAN handle an unlimited number of comments or do we need to enforce a max?

Reply:
Since this is based on the number of tags you can have, the limit maybe quite large. I can't find anything published.

Reply:
maximum tags is 10,000: https://knowledgebase.paloaltonetworks.com/KCSArticleDetail?id=kA10g000000PO8vCAG&lang=en_US%E2%80%A9

Reply:
That sounds good to me. We'll follow up with the rest of the configurable limits.